From 8a2fe5a35baf671dafc695a30777d5e6cc648dcf Mon Sep 17 00:00:00 2001
From: anitha3x <anithax.h.chandrasekar@intel.com>
Date: Thu, 28 Jun 2018 14:35:59 +0530
Subject: [PATCH 5/6] WBS eSco parameters updated for intel BT controller

issue :
- eSco link for WBS failed
- The HCI_Enhanced_Setup_Synchronous_Connection returned
  "Invalid HCI command parameters" from controller
fix:
- eSco mSBC params changed as per Intel connectivity
- considered T1 setting params for eSco as per HFP 1.6
Testing :
- HFP call initiating from DUT to phone
- HFP call initiating from phone to DUT

Change-Id: I2f8f38d95cf13be40cf4b18b471a547d96c7e4cb
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-65939
Signed-off-by: anitha3x <anithax.h.chandrasekar@intel.com>
---
 device/src/esco_parameters.cc | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/device/src/esco_parameters.cc b/device/src/esco_parameters.cc
index f38e56f..7b72016 100644
--- a/device/src/esco_parameters.cc
+++ b/device/src/esco_parameters.cc
@@ -74,20 +74,20 @@ static const enh_esco_params_t default_esco_parameters[ESCO_NUM_CODECS] = {
     {.transmit_bandwidth = TXRX_64KBITS_RATE,
      .receive_bandwidth = TXRX_64KBITS_RATE,
      .transmit_coding_format = {.coding_format = ESCO_CODING_FORMAT_MSBC,
-                                .company_id = 0x0000,
+                                .company_id = 0x0002, /* Intel Corp.*/
                                 .vendor_specific_codec_id = 0x0000},
      .receive_coding_format = {.coding_format = ESCO_CODING_FORMAT_MSBC,
-                               .company_id = 0x0000,
+                               .company_id = 0x0002, /* Intel Corp.*/
                                .vendor_specific_codec_id = 0x0000},
      .transmit_codec_frame_size = 60,
      .receive_codec_frame_size = 60,
      .input_bandwidth = INPUT_OUTPUT_128K_RATE,
      .output_bandwidth = INPUT_OUTPUT_128K_RATE,
      .input_coding_format = {.coding_format = ESCO_CODING_FORMAT_LINEAR,
-                             .company_id = 0x0000,
+                             .company_id = 0x0002, /* Intel Corp.*/
                              .vendor_specific_codec_id = 0x0000},
      .output_coding_format = {.coding_format = ESCO_CODING_FORMAT_LINEAR,
-                              .company_id = 0x0000,
+                              .company_id = 0x0002, /* Intel Corp.*/
                               .vendor_specific_codec_id = 0x0000},
      .input_coded_data_size = 16,
      .output_coded_data_size = 16,
@@ -97,8 +97,8 @@ static const enh_esco_params_t default_esco_parameters[ESCO_NUM_CODECS] = {
      .output_pcm_payload_msb_position = 0,
      .input_data_path = ESCO_DATA_PATH_PCM,
      .output_data_path = ESCO_DATA_PATH_PCM,
-     .input_transport_unit_size = 0x00,
-     .output_transport_unit_size = 0x00,
+     .input_transport_unit_size = 0x10,
+     .output_transport_unit_size = 0x10,
      .max_latency_ms = 8,
      .packet_types =
          (ESCO_PKT_TYPES_MASK_EV3 | ESCO_PKT_TYPES_MASK_NO_3_EV3 |
-- 
1.9.1

